Client/Frontend/Widgets/PhotonActionSheet/PageActionMenu.swift

```diff
@@ -216,7 +217,7 @@ extension PhotonActionSheetProtocol {
             commonActions.insert(findInPageAction, at: 0)
         }
 
-        if (profile.prefs.boolForKey(PrefsKeys.ShowNewTabToolbarButton) ?? false && tab.readerModeAvailableOrActive) {
+        if shouldShowNewTabButton && tab.readerModeAvailableOrActive {
```

Comment:
additional variable which now takes into account for showing or hiding tab button and logic is moved to BVC